js/views/currencyListVw.js

```diff
@@ -29,11 +29,12 @@ module.exports = Backbone.View.extend({
     orderedCurrencies.unshift({code: "BTC", currency: "Bitcoin", currencyUnits: "4"});
     this.chooseCurrencies = new chooseCurrenciesCollection(orderedCurrencies);
     this.subViews = [];
+
     //use default currency to return list of supported currencies
     getBTPrice("USD", function (btAve, currencyList) {
-      "use strict";
```

Comment:
Fixed. I was doing some debugging and I'd left an extraneous function call in there.